src/math-equation.c

```diff
@@ -971,6 +971,24 @@ math_equation_set(MathEquation *equation, const gchar *text)
     clear_ans(equation, FALSE);
 }
 
+void
+math_equation_set_with_history(MathEquation *equation, const gchar *text)
```

Comment:
At first glance the function `math_equation_set_with_history` may be static because it is only used in the math-equation.c file.

Reply:
```diff
diff --git a/src/math-equation.c b/src/math-equation.c
index 167e0ec..c739197 100644
--- a/src/math-equation.c
+++ b/src/math-equation.c
@@ -971,7 +971,7 @@ math_equation_set(MathEquation *equation, const gchar *text)
     clear_ans(equation, FALSE);
 }
 
-void
+static void
 math_equation_set_with_history(MathEquation *equation, const gchar *text)
 {
     MathEquationState *state;
diff --git a/src/math-equation.h b/src/math-equation.h
index c62564d..954334c 100644
--- a/src/math-equation.h
+++ b/src/math-equation.h
@@ -108,7 +108,6 @@ void math_equation_redo(MathEquation *equation);
 void math_equation_store(MathEquation *equation, const gchar *name);
 void math_equation_recall(MathEquation *equation, const gchar *name);
 void math_equation_set(MathEquation *equation, const gchar *text);
-void math_equation_set_with_history(MathEquation *equation, const gchar *text);
 void math_equation_set_number(MathEquation *equation, const MPNumber *x);
 void math_equation_insert(MathEquation *equation, const gchar *text);
 void math_equation_insert_digit(MathEquation *equation, guint digit);
```

Reply:
@rbuj done